src/runtime_src/core/common/drv/kds_core.c

```diff
@@ -577,6 +578,14 @@ int kds_add_context(struct kds_sched *kds, struct kds_client *client,
 			return -EINVAL;
 		}
 		++client->virt_cu_ref;
+		/* a special handling for m2m cu :( */
```

Comment:
should this special handling be done when the client reserves virtual CU for the first time only?

Reply:
Sure. we can.